rust/Cargo.toml.in

```diff
@@ -28,6 +28,7 @@ num = "0.2"
 num-derive = "0.2"
 num-traits = "0.2"
 widestring = "0.4"
+md5 = "0.7.0"
```

Comment:
@jasonish opinions on this ?

performance discussed here https://github.com/OISF/suricata/pull/4974#discussion_r432895304

If we choose to go with this, should we create a ticket to get rid of libnss and move to Rust crypto crates ?